### DISCUSSION TITLE
Problem with time stamp and audio size

### DISCUSSION THREAD

**Discussion by titusfx** [openai/whisper#124] [Q&A] [answered]
I've an audio that is `9 seconds long`, whisper returns a `transcription with 34 seconds long`. 

The audio has 4 phrases.
The audio is `mp3` file and the result is on the `txt` file (`json` file)

[audio_example](https://user-images.githubusercontent.com/1366004/192151886-76448074-e7b2-4d1c-add8-636e64247916.mp4)
[whisper_result](https://github.com/openai/whisper/files/9641141/whisper_result.txt)

**Comment by jongwook** [comment, marked as answer]
We could add a post-processing step so that the timestamps are upper-bounded by the audio length. This wasn't my priority because the timestamps tend to become more accurate with the larger models and a prolonged last timestamp didn't hurt much practically.

**Reply by kmk2018** [reply]
Hello, 
This problem can occur within an utterance if Whisper breaks the utterance into multiple phrases, not just the final end time. For example this is from a 11.09s file with 4 phrases where the 3rd and 4th phrase have the wrong timings:

["cand", [{"id": 0, "seek": 0, **"start": 0.0, "end": 4.6000000000000005,** "text": ...}, {"id": 1, "seek": 0, **"start": 4.6000000000000005, "end": 8.64,** "text": ...}, {"id": 2, "seek": 864, **"start": 8.64, "end": 14.66,** "text": "...}, {"id": 3, "seek": 864, **"start": 14.66, "end": 34.36,** "text": ...}]], 

FYI the input was a .wav file

**Reply by Pricer187** [reply]
> We could add a post-processing step so that the timestamps are upper-bounded by the audio length. This wasn't my priority because the timestamps tend to become more accurate with the larger models and a prolonged last timestamp didn't hurt much practically.
